demos/src/demo.scss

```diff
@@ -8,18 +8,5 @@ html {
 
 body {
 	margin: oGridGutter();
-}
-
-.demo-real-life {
-	padding: 0;
-
-	body {
-		margin: 0;
-	}
-
-	h2,
-	h3 {
-		@include oFormsGroup; // Indent
-		margin-bottom: 1em; // Set smaller margin
-	}
+	background-color: #fff1e0;
```

Comment:
This is only the demo css, so I think it's ok here. This isn't actually part of the component.